db/migrate/20240514183639_add_original_cross_core_order_to_orders.rb

```diff
@@ -0,0 +1,17 @@
+class AddOriginalCrossCoreOrderToOrders < ActiveRecord::Migration[7.0]
+  def up
+    add_column :orders, :original_cross_core_order, :boolean, default: false, null: false
+
+    Projects::Project.find_each do |project|
+      next unless project.orders.any?
+
+      original_order = project.orders.where(facility_id: project.facility_id).first
+
+      original_order&.update!(original_cross_core_order: true)
```

Comment:
I'm not sure we want to raise an exception here, maybe just log the project ID and error instead?